### DISCUSSION TITLE
Extrusion width/ Flow ratio on First Layer

### DISCUSSION THREAD

**Discussion by UD551** [supermerill/SuperSlicer#1442] [Q&A]
Hello all,
I was facing an issue when trying to print large extrusion width on the first layer. I am currently using a 0.4mm nozzle on the Prusa printer and I am trying to figure out what is the physical limit to my extrusion width that I am able to achieve on the first layer. When I tried an extrusion width of 1.2mm for the first layer, the first layer turned out to be (under extruded) since the extrusion lines didn't adhere to each other properly. So, what I did to fix this, is I increased the flow rate of the first layer to 150% (Width& Flow --> First Layer Flow Ratio). When I did that, the print turned out to be pretty good. 

However, shouldn't the printer/ super slicer account for that? In other words, shouldn't the printer know by itself that it needs to over extrude filament to account for the large extrusion widths used? in order for it to adhere correctly and close the gaps.  
In addition, what is the physical limit that I might be able to go up to when increasing my extrusion widths for the first layer? what is my maximum extrusion widths I can achieve before breaking the printer? 

Your help is greatly appreciated. Thank you for your time. 
I hope to hear back from you on that matter.

**Comment by 5axes** [comment]
Usualy a limit recommanded for the extrusion width is around (` nozzle diameter * 1.7`)

**Reply by UD551** [reply]
I don't think this is the case because we are already printing at 0.8mm width for the first layer and it's turning out to be good without changing the flow ratio. However, my issue happens for widths larger than 0.8. For example, an extrusion width of 1mm needs a flow ratio of 120%. This means the superslicer / printer is not accounting for the increase in extrusion widths.

**Comment by supermerill** [comment]
See https://youtu.be/0xRtypDjNvI?t=301
It's not compensated yet.
Have to create a table in filament (or extruder, printer?) profile, and associated calibration to fill it.
